CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 31.2
+- show memory profiler legend with proper line chart color or line style
+- prevent the (IntelliJ) New Project wizard from completing when there is no Flutter SDK
+- fix a race condition causing unexpected conditions in attach
+- added control of RSS display to memory profiler
+- when running the flutter doctor command, use the -v flag
+- add https_proxy to the readme
```

Comment:
I'd maybe drop this since it's not plugin-facing.

Reply:
Done